lib/struct.js

```diff
@@ -21,7 +21,7 @@ var _lodash = require("lodash");
 var each = _lodash.each;
 var map = _lodash.map;
 var isUndefined = _lodash.isUndefined;
-var zipObject = _lodash.zipObject;
+var fromPairs = _lodash.fromPairs;
```

Comment:
Are you sure both functions are working in the same way? I'm checking the docs and they seem to different:
* [`fromPairs`](https://lodash.com/docs/4.17.10#fromPairs)
```js
_.fromPairs([['a', 1], ['b', 2]]);
// => { 'a': 1, 'b': 2 }
```
* [`zipObject`](https://lodash.com/docs/4.17.10#zipObject)
```js
_.zipObject(['a', 'b'], [1, 2]);
// => { 'a': 1, 'b': 2 }
```

Reply:
It's been a while since I wrote this code, but as I recall, the code was using _.zipObject in Lodash 3 in a way that is now handled by _.fromPairs in Lodash 4.

This is the code (before my change): 

https://github.com/stellar/js-xdr/blob/master/src/struct.js#L12-L20

```
  static read(io) {
    let fields = map(this._fields, field => {
      let [name, type] = field;
      let value = type.read(io);
      return [name, value];
    });

    return new this(zipObject(fields));
  }
```

So it makes a list of [ name, value ] pairs and zips it into an array of objects `{ [name]: value }`.

zipObject in Lodash 3 did that....

https://lodash.com/docs/3.10.1#zipObject

```
_.zipObject([['fred', 30], ['barney', 40]]);
// => { 'fred': 30, 'barney': 40 }
 
_.zipObject(['fred', 'barney'], [30, 40]);
// => { 'fred': 30, 'barney': 40 }
```

...but that particular function signature got split off into fromPairs in Lodash 4.

https://lodash.com/docs/4.17.10#fromPairs

```
_.fromPairs([['a', 1], ['b', 2]]);
// => { 'a': 1, 'b': 2 }
```

I don't remember for sure but I think I surfaced this issue in the unit tests. The tests should fail in the previous commit and pass in the latter one.

Reply:
Oh man! I missed one pair of `[]`... 🤦‍♂️ Sorry, looks good!